src/context/trace_context.rs

```diff
@@ -182,8 +189,13 @@ impl TracingContext {
         self.with_active_span_stack(|stack| stack.last().map(f))
     }
 
-    // TODO Using for capture and continued.
-    #[allow(dead_code)]
+    pub(crate) fn with_active_span_mut<T>(
```

Comment:
Could you share what does this method mean?

Reply:
This method accepts a closure that modifies the top element of the active span stack. Because `Arc` is used, reference counting has no mutable method, so `Mutex` is required for internal mutability, and `with_*` methods can easily handle `try_lock` and drop, , shielding these details.